### PR TITLE
Added start game sound 3-2-1 in the begining of the game.

### PR DESCRIPTION
**Pull request by h4rdmol**
https://trello.com/c/dMLSHoJH/487-3-2-1-countdown-timer-sound
Sound used: https://freesound.org/people/ImATaco/sounds/629028/

image

image

image

https://user-images.githubusercontent.com/5227233/208278372-ca5d8877-fb60-4d37-aabf-b6a4c85281ab.MP4

### CODE REVIEW

File: Source/Bomber/Private/GameFramework/MyGameStateBase.cpp [hunk 1]
```diff
@@ -81,6 +81,7 @@ void AMyGameStateBase::ApplyGameState()
 {
 	if (CurrentGameStateInternal == ECGS::GameStarting)
 	{
+		USoundsManager::Get().PlayStartGameCountdownSFX();
```
Comment (JanSeliv) [inline Source/Bomber/Private/GameFramework/MyGameStateBase.cpp:84]: Please better move it inside AMyGameStateBase::TriggerCountdowns(). It's the function that triggers this coundown, so if someone call it somewhere manually (like custom mode, where could be manual TriggerCountdowns call), then this sound is also would be played automatically without bug

File: Source/Bomber/Public/Globals/SoundsDataAsset.h [hunk 2]
```diff
@@ -140,4 +149,4 @@ class BOMBER_API USoundsDataAsset final : public UDataAsset
 	 * @see USoundsDataAsset::SoundsManagerClass */
 	UPROPERTY(Transient)
 	mutable TObjectPtr<USoundsManager> SoundsManager = nullptr;
-};
```
Comment (JanSeliv) [inline Source/Bomber/Public/Globals/SoundsDataAsset.h:143]: What is the line removal? Please get rid of this change

File: Source/Bomber/Private/SoundsManager.cpp [hunk 3]
```diff
@@ -129,6 +129,19 @@ void USoundsManager::PlayEndGameCountdownSFX()
 	}
 }
 
+void USoundsManager::PlayStartGameCountdownSFX()
```
Comment (JanSeliv) [inline Source/Bomber/Private/SoundsManager.cpp:132]: is minor, but please add comment above like in every function in .cpp

**Comment by h4rdmol** [comment]
image

**Review by JanSeliv — CHANGES_REQUESTED** [review]
Thank you, left comments

**Review by JanSeliv — CHANGES_REQUESTED** [review]
Left new comments

File: Source/Bomber/Private/SoundsManager.cpp (hunk 3 above)
Comment (JanSeliv) [reply Source/Bomber/Private/SoundsManager.cpp:133]: javadoc format /*\* */ is only used in header while in cpp regular comments //

**Review by JanSeliv — CHANGES_REQUESTED** [review]
One left comment

**Pull request merged**